src/libAtomVM/nifs.c

```diff
@@ -32,6 +32,7 @@
 #include <time.h>
```

Comment:
here there is lot of clang-format noise.

Reply:
What would you suggest? I can clean up the `static const char *const` lines using `ATOM_STR()`, but clang-format-16 takes issue with our `static const struct Nif` formatting, as well as a few other differences from clang-format-13.